pages.ru/common/head.md

```diff
@@ -0,0 +1,20 @@
+# head
+
+> Выводит первую часть файлов.
+> Больше информации: <https://www.gnu.org/software/coreutils/head>.
+
+- Выводит первые несколько строк из файла:
```

Comment:
```suggestion
- Вывести первые несколько строк из файла:
```

Everything is okay, just one suggestion. Also change other descriptions as this one. ;)

Reply:
Changed